orchestrator-salt/src/main/resources/salt/salt/java/init.sls

```diff
@@ -1,3 +1,17 @@
+/opt/salt/scripts/set_default_java_version.sh:
+  file.managed:
+    - source:
+        - salt://java/scripts/set_default_java_version.sh
```

Comment:
the state file and the script name also should reflect, that this is setting java to the 11 version. 
so it would be better to have a state called java11.sls and also the script name like set_default_java_version_to_11

Reply:
Its not about java 11, it is about changing the default java version to this or that. This script can be extended with magic command for 17 or whatever version later in case of any backward incompatibility.